src/main/kotlin/com/hibob/academy/employeeFeedback/service/FeedbackService.kt

```diff
@@ -0,0 +1,70 @@
+package com.hibob.academy.employeeFeedback.service
+
+import com.hibob.academy.employeeFeedback.dao.*
+import jakarta.ws.rs.BadRequestException
+import org.springframework.stereotype.Service
+
+@Service
+class FeedbackService(private val feedbackDao: FeedbackDao) {
+    fun feedbackLengthValidation(feedback: String): Boolean {
+        val minCharacters = 30
+        val maxCharacters = 500
+        return !(feedback.length < minCharacters || feedback.length > maxCharacters)
+    }
+
+    fun anonymityStatusValidation(employeeId: Int?): Int{
+        return employeeId ?: throw Exception("Can't check status of anonymous feedback")
```

Comment:
why generic exception but in the method below it's BadRequest?
They should be the same kind of exception because it's the same scenario.